resources/ec2-internet-gateway-attachments.go

```diff
@@ -10,11 +10,12 @@ import (
 )
 
 type EC2InternetGatewayAttachment struct {
-	svc     *ec2.EC2
-	vpcId   *string
-	vpcTags []*ec2.Tag
-	igwId   *string
-	igwTags []*ec2.Tag
+	svc         *ec2.EC2
+	vpcId       *string
+	vpcTags     []*ec2.Tag
+	igwId       *string
+	igwTags     []*ec2.Tag
+	defVpcAssoc bool
```

Comment:
I would simply call this default VPC. Also to follow the naming convention, `VPC` should be spelled uppercase.

```suggestion
	defaultVPC bool
```